src/System.Reflection.Primitives/src/System/Reflection/CallingConventions.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+////////////////////////////////////////////////////////////////////////////////
+////////////////////////////////////////////////////////////////////////////////
+//
+// CallingConventions is a set of Bits representing the calling conventions
+// 
+
+//    in the system.
+//
+// <EMAIL>Author: meichint</EMAIL>
+// Date: Aug 99
+//
+
+using System.Runtime.InteropServices;
+using System;
```

Comment:
Try to remove unused using statements in these files. I think all of them have `using System;` at the very least which his unnecessary.